### ISSUE TITLE
bought price is not correct, worst case.

### ISSUE THREAD

**Issue by Cryptomi** [taniman/profit-trailer#1782]
iam using pt since last year and run between 6-12 bots. Every bot have his own vps.
But since 2.0.  there are a lot of manual bought coins showing me wrong price, also the bot thinks that the coin is in profit and sell it in loss just a couple seconds after buying the coin. 
Deleting jason file and backup ore setup a fresh new bot dosnt help. this is really worst case.

**Comment by Cryptomi** [comment]
![bildschirmfoto 2018-05-04 um 15 50 57](https://user-images.githubusercontent.com/35530935/39631633-fffefc72-4fb2-11e8-88c6-f6efe82a842a.png)
 
a lot off sells like this, but not on every coin. some coins are correct price and some coins are incorrect.

**Comment by Utone** [comment]
year old issue PMed about basically set to watch mode make note of your average bought price before selling an earlier DCA that is going baggy soon too soon to save it you sell with profit right?
Well you will sell quickly but not at a profit (the remaining bag) as it suddenly idk how restarted that math is it thinks its in profit and sells....

Another fine fuck up probably still in there (and I also Pmed but no replay and lets say i don't like the guy very much after all the trouble i went trough for him... ) BUT THIS IS FOR THE PEOPLE

I had a very nice big dca with 88%profit (over 4.5k in profit) to see it dissapear only after at least 102 minutes to pop back up... after a partial fill... It was recalculation he AVERAGE BOUGHT PRICE OF A PAIR WHICH HAD NOT CHANGED in this regard (after disabling trailing profit, finally undrrstanding what it was doing i typed a panic sell for a measly 90 bucks... made think 2k instead of the 4.5 it should have been able to process much more intelligently 

(setting min volume 125 doen't really work in dumps and neither does trailing profit which you should  turn off ASAP in this scenario 

(RSI Sell at level 83 or something would be good dirty fix (BUT for that to work bit needs to allow mid sales or before that very high level is met as we use it as a safety feature to not get in dumps)

 I have seen people that went in and out of the pump whithout selling because of it..)

**Comment by SgtBatten** [comment]
@utone please post in the right issue thread. This one is about manual buying not partial sells causing incorrect averages.

**Comment by taniman** [comment]
missing logs and everything. create a new issue according to the issues template
